src/man/sssd-idp.5.xml

```diff
@@ -0,0 +1,218 @@
+                    <term>idp_id_scope (string)</term>
+                    <listitem>
+                        <para>
+                            Scope required for looking up user and group
+                            attributes with the REST API. The scopes are used
+                            by the server to determine which attributes/claims
+                            are returned to the caller.
+                        </para>
+                        <para>
+                            Default: Not set
+                        </para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>idp_auth_scope (string)</term>
+                    <listitem>
+                        <para>
+                            Scope required during authentication. The scopes are
+                            used by the server to determine which
+                            attributes/claims are returned to the caller.
+                        </para>
+                        <para>
+                            Currently the tokens returned during user
+                            authentication are not used for other purposes
+                            hence the only important claim is the subject
+                            identifier 'sub' which is used to check if the
+                            authenticated user is the one trying to log in.
+                            This might change in future.
+                        </para>
+                        <para>
+                            Default: Not set
+                        </para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>idp_request_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Timeout in seconds for an individual request to
+                            the IdP.
+                        </para>
+                        <para>
+                            Default: 10
+                        </para>
+                    </listitem>
+                </varlistentry>
+            </variablelist>
+        </para>
+    </refsect1>
+
+    <refsect1 id='example'>
+        <title>EXAMPLE</title>
+        <para>
+<programlisting>
+[domain/entra_id]
+id_provider = idp
+idp_type = entra_id
+idp_client_id = 12345678-abcd-0101-efef-ba9876543210
+idp_client_secret = YOUR-CLIENT-SCERET
+idp_token_endpoint = https://login.microsoftonline.com/TENNANT-ID/oauth2/v2.0/token
+idp_userinfo_endpoint = https://graph.microsoft.com/v1.0/me
+idp_device_auth_endpoint = https://login.microsoftonline.com/TENNANT-ID/oauth2/v2.0/devicecode
+idp_id_scope = https%3A%2F%2Fgraph.microsoft.com%2F.default
+idp_auth_scope = openid profile email
+</programlisting>
+<programlisting>
+[domain/keycloak]
+idp_type = keycloak:https://master.keycloak.test:8443/auth/admin/realms/master/
```

Comment:
Description of `idp_type` option doesn't describe this syntax - 'type:URL'?
Is URL really required here?